recipe/meta.yaml

```diff
@@ -297,12 +300,20 @@ outputs:
         Activation for Intel® oneAPI DPC++/C++ Compiler.
         This package is a repackaged set of binaries obtained directly from Intel's Anaconda.org channel.
     test:
+      requires:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      files:
+        - simple.cpp
       commands:
         - ls -A1 ${PREFIX}/bin/*         # [unix]
         - ls -A1 ${PREFIX}/include/*     # [unix]
         - ls -A1 ${PREFIX}/lib/*         # [unix]
         - dir %PREFIX%\Library\bin\*  # [win]
         - dir %PREFIX%\Library\lib\*  # [win]
+        - dpcpp --gcc-toolchain=$PREFIX --sysroot=$PREFIX/$BUILD/sysroot -target $BUILD ${LDFLAGS} ${CXXFLAGS} simple.cpp -lpthread -o simple.exe  # [unix]
+        # Need for -I option is a bug in compiler activation script. Can be removed when fixed
+        - dpcpp simple.cpp -I%PREFIX%\include -o simple.exe  # [win]
```

Comment:
SYCL include files should go into `%PREFIX%\Library\include` instead of `%PREFIX%\include`

Reply:
Yes, understood. This fix will be applied in the compiler package in a future release at which time I will remove this kludge.